server/src/metadata/migration-runner/migration-runner.service.ts

```diff
@@ -113,13 +113,23 @@ export class MigrationGeneratorService {
             name: 'id',
             type: 'uuid',
             isPrimary: true,
-            default: 'uuid_generate_v4()',
+            default: 'public.uuid_generate_v4()',
```

Comment:
This part was broken, from my understanding adding the uuid extension does not install the function on every schemas but on public only so you have to specify it otherwise typeorm will fail